infra/src/main/resources/db/migration/repeatable/R__04.06_track_number_in_layout_context.sql

```diff
@@ -0,0 +1,57 @@
+drop view if exists layout.track_number_publication_view;
```

Comment:
Ymmärrettävää että repeatableissa halutaan varmistua siitä että migraatio tekee varmasti juuri sen kaiken mitä halutaan, mutta tämä sama droppi tehdään jo tuolla `R__04.01`-migraatiossa

Reply:
Juuri tuo rivi tässä tiedostossa oli copypasten jätöksiä, poistan sen.

Uudelleennimeämistä luulin tätä koodia kirjoittaessani vaikeammaksi asiaksi, mutta eihän siinä tarvita muuta kuin konffata Flyway olemaan välittämättä "puuttuvista" repeatable-migroista: https://documentation.red-gate.com/flyway/flyway-cli-and-api/configuration/parameters/flyway/ignore-migration-patterns